packages/forma-36-react-components/package.json

```diff
@@ -81,9 +81,7 @@
     "postcss-loader": "^3.0.0",
     "postcss-preset-env": "^6.7.0",
     "postcss-url": "^8.0.0",
-    "react": "^16.8.3",
     "react-docgen-typescript-loader": "3.1.0",
-    "react-dom": "^16.8.3",
```

Comment:
I think we need only one, right?! Otherwise we would have two reacts when developing (That was how it was before this PR, dev was 16.8 and peer was 16.3 👀 )

with react only in peerDependency, this package will get react from the node_modules in the root of the repo if I'm not mistaken

Reply:
Usually the peer dependencies aren't installed, so you'd end up actually missing React when developing locally. I don't have a React dependency in my root node_modules, do you? 🤔